source/SquashApp.mc

```diff
@@ -1,41 +1,65 @@
+using Toybox.System;
 using Toybox.Application as App;
 using Toybox.WatchUi as Ui;
 using Toybox.Sensor as Snsr;
 
 //! Class that represents the Squash Application
 class SquashApp extends App.AppBase {
 
-	//! Object that contains the data that will
-	//! be displayed on screen
-	hidden var dataTracker;
+    //! Object that contains the data that will
+    //! be displayed on screen
+    hidden var dataTracker;
 
-	//! Constructor
+    //! Points to win a set
+    hidden var setMaxScore = 99;

```

Comment:
These two variables should not belong to the App object. I will change this later to improve encapsulation and locality of change.